### PR TITLE
Timeouttable

### PR DESCRIPTION
**Pull request by AlexanderVladimirov9090**

### CODE REVIEW

File: arrayaddremove/src/main/java/com/clouway/fixedsizelist/FixedList.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+     * @throws ListFullException
+     */
+    public synchronized void add(Object item) throws ListFullException {
+
+        if (index < items.length) {
+            items[index] = item;
+            index++;
+        } else {
+            throw new ListFullException();
+        }
```
Comment (EmilGeorgiev) [inline arrayaddremove/src/main/java/com/clouway/fixedsizelist/FixedList.java]:
```java
if (index < items.length) {
     throw new ListFullException();            

} 

items[index] = item;
index++;
```

File: arrayaddremove/src/main/java/com/clouway/fixedsizelist/FixedList.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    /**
+     * Removes object from array.
+     *
+     * @throws ListEmptyException
+     */
+    public synchronized void remove() throws ListEmptyException {
+
+        if (items[index - 1] == null) {
+            throw new ListEmptyException();
+        } else {
```
Comment (EmilGeorgiev) [inline arrayaddremove/src/main/java/com/clouway/fixedsizelist/FixedList.java]: avoid ```else```

File: arrayaddremove/src/main/java/com/clouway/fixedsizelist/ListFullException.java [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+ *
+ * @author Alexander Vladimirov
+ *         <alexandervladimirov1902@gmail.com>
+ */
+
+public class ListFullException extends RuntimeException{
+
+    public ListFullException() {
+        super("Can not add list is full");
+    }
```
Comment (EmilGeorgiev) [inline arrayaddremove/src/main/java/com/clouway/fixedsizelist/ListFullException.java]:
```java
public ListFullException(String message) {
      super(mesage);
}
```

File: arrayaddremove/src/main/java/com/clouway/fixedsizelist/FixedList.java (hunk 1 above)
Comment (AlexanderVladimirov9090) [inline arrayaddremove/src/main/java/com/clouway/fixedsizelist/FixedList.java]: Done.

File: arrayaddremove/src/main/java/com/clouway/fixedsizelist/FixedList.java (hunk 2 above)
Comment (AlexanderVladimirov9090) [inline arrayaddremove/src/main/java/com/clouway/fixedsizelist/FixedList.java]: Done.

**Comment by AlexanderVladimirov9090** [comment]
Done.

File: arrayaddremove/src/main/java/com/clouway/fixedsizelist/ListFullException.java (hunk 3 above)
Comment (AlexanderVladimirov9090) [inline arrayaddremove/src/main/java/com/clouway/fixedsizelist/ListFullException.java]: Done.

**Comment by EmilGeorgiev** [comment]
LGTM

**Pull request merged**